bizflycloud/resource_bizflycloud_server_test.go

```diff
@@ -56,6 +56,12 @@ func TestAccBizFlyCloudServer_Basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet("bizflycloud_server.foobar", "created_at"),
 				),
 			},
+			{
+				ResourceName:            "bizflycloud_server.foobar",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{""},
```

Comment:
May need to add some ignore here to pass the test since API doesn't return all the fields in the terraform state.